### ISSUE TITLE
Error run report allure

### ISSUE THREAD

**Issue by Smolevich** [allure-framework/allure-bamboo#131]
Allure plugin version: 1.10
Bamboo version: 7.0.2 build 70015
Allure agent version: 2.7.0
Error message: 	
```
simple 07-May-2020 18:14:56	Starting task 'allure report' of type 'io.qameta.allure.allure-bamboo:allureReportTask'
error	07-May-2020 18:14:56	Error occurred while running Task 'allure report(6)' of type io.qameta.allure.allure-bamboo:allureReportTask.
error	07-May-2020 18:14:56	java.lang.NoSuchMethodError: com.atlassian.bamboo.build.logger.BuildLogger.addBuildLogHeader(Ljava/lang/String;Z)Ljava/lang/String;
error	07-May-2020 18:14:56		at io.qameta.allure.bamboo.AllureReportTask.execute(AllureReportTask.java:23)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.task.TaskExecutorImpl.lambda$executeTasks$2(TaskExecutorImpl.java:321)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.task.TaskExecutorImpl.executeTaskWithPrePostActions(TaskExecutorImpl.java:256)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.task.TaskExecutorImpl.executeTasks(TaskExecutorImpl.java:321)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.task.TaskExecutorImpl.execute(TaskExecutorImpl.java:124)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.build.pipeline.tasks.ExecuteBuildTask.call(ExecuteBuildTask.java:73)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.v2.build.agent.DefaultBuildAgent.executeBuildPhase(DefaultBuildAgent.java:184)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.v2.build.agent.DefaultBuildAgent.build(DefaultBuildAgent.java:156)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.v2.build.agent.BuildAgentControllerImpl.lambda$waitAndPerformBuild$0(BuildAgentControllerImpl.java:130)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.variable.CustomVariableContextImpl.withVariableSubstitutor(CustomVariableContextImpl.java:118)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.v2.build.agent.BuildAgentControllerImpl.waitAndPerformBuild(BuildAgentControllerImpl.java:124)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.v2.build.agent.DefaultBuildAgent.lambda$start$0(DefaultBuildAgent.java:108)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.utils.BambooRunnables$1.run(BambooRunnables.java:48)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.security.ImpersonationHelper.runWith(ImpersonationHelper.java:26)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.security.ImpersonationHelper.runWithSystemAuthority(ImpersonationHelper.java:17)
error	07-May-2020 18:14:56		at com.atlassian.bamboo.security.ImpersonationHelper$1.run(ImpersonationHelper.java:41)
error	07-May-2020 18:14:56		at java.lang.Thread.run(Thread.java:748)
```

**Comment by beteshnik** [comment]
We had the same trouble. As turned out the large report size caused the problem. The solution is to expand size in nginx bamboo settings.

**Comment by jappenzesr** [comment]
We have the exact same issue in Bamboo 7.0.4. The plugin needs to be properly migrated to Bamboo 7.x, the API method signatures changed. 

@beteshnik I don't know how nginx settings could solve this, the only way to not run into this issue is that the Allure task is not even executed. What was the exact error message that you got?

**Comment by vrosansys** [comment]
We have the exact same issue in Bamboo 7.1.2

**Comment by shapeshifta78** [comment]
Same here, will you fix it?

**Comment by rob-swift-ln** [comment]
We are getting this error with the following:

Allure plugin version: 1.14.0
Bamboo version: 8.2.1 build 80206 - 08 Apr 22
Allure agent version: 2.17.2

I saw the same error was present for Issue 196.  In that issue it was suggested to set "Allure binary base url" to https://github.com/allure-framework/allure2/releases/download/ for versions >= 2.13.1, which we have done.

I was thinking that this would have been resolved between versions 2.13 & 2.14 of the plugin.  We are finding that the report does get created, but the job is failing with the exception:

```
simple	02-Sep-2022 10:33:07	Starting task 'Allure Report' of type 'io.qameta.allure.allure-bamboo:allureReportTask'
error	02-Sep-2022 10:33:07	Error occurred while running Task 'Allure Report(3)' of type io.qameta.allure.allure-bamboo:allureReportTask.
error	02-Sep-2022 10:33:07	java.lang.NoSuchMethodError: com.atlassian.bamboo.build.logger.BuildLogger.addBuildLogHeader(Ljava/lang/String;Z)Ljava/lang/String;
error	02-Sep-2022 10:33:07		at io.qameta.allure.bamboo.AllureReportTask.execute(AllureReportTask.java:23)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.task.TaskExecutorImpl.lambda$executeTasks$2(TaskExecutorImpl.java:321)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.task.TaskExecutorImpl.executeTaskWithPrePostActions(TaskExecutorImpl.java:258)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.task.TaskExecutorImpl.executeTasks(TaskExecutorImpl.java:321)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.task.TaskExecutorImpl.execute(TaskExecutorImpl.java:118)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.build.pipeline.tasks.ExecuteBuildTask.call(ExecuteBuildTask.java:73)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.v2.build.agent.DefaultBuildAgent.executeBuildPhase(DefaultBuildAgent.java:205)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.v2.build.agent.DefaultBuildAgent.build(DefaultBuildAgent.java:177)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.v2.build.agent.BuildAgentControllerImpl.lambda$waitAndPerformBuild$0(BuildAgentControllerImpl.java:169)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.variable.CustomVariableContextImpl.withVariableSubstitutor(CustomVariableContextImpl.java:118)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.v2.build.agent.BuildAgentControllerImpl.waitAndPerformBuild(BuildAgentControllerImpl.java:162)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.v2.build.agent.DefaultBuildAgent.lambda$start$0(DefaultBuildAgent.java:122)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.utils.BambooRunnables$1.run(BambooRunnables.java:48)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.security.ImpersonationHelper.runWith(ImpersonationHelper.java:26)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.security.ImpersonationHelper.runWithSystemAuthority(ImpersonationHelper.java:17)
error	02-Sep-2022 10:33:07		at com.atlassian.bamboo.security.ImpersonationHelper$1.run(ImpersonationHelper.java:41)
error	02-Sep-2022 10:33:07		at java.lang.Thread.run(Thread.java:750)
```

**Comment by Vlad-iodine** [comment]
Still very much happening. Any updates or resolution ETA?
[Atlassian Bamboo] version 7.2.7 build 70214
Allure for Bamboo 1.14.0
cypress-allure-plugin": "^2.28.5

**Comment by vbragin** [comment]
was fixed with #270
